documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -175,7 +178,7 @@ EOF
 ----
 <1> Allowed values are `pod` and `multus`.
 <2> You can use either the `id` _or_ the `name` parameter to specify the source network.
-<3> Specify the VMware network MOR, the {rhv-short} network UUID, or the {osp} network UUID.
+<3> Specify the VMware network MOR, the {rhv-short} network UUID, the {osp} network UUID, or the OVA network UUID.
```

Comment:
@RichardHoch 
Maybe you should say what UUID and MOR stand for when these acronyms first appear in the text - Universal Unique ID and Managed Object Reference. Or is it obvious to the reader?  Up to you.